packages/wrangler/src/constellation/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { listModels } from "./listModel";
+import { listProjects } from "./listProject";
+import type { Config } from "../config";
+import type { Project, Model } from "./types";
+
+export const constellationBetaWarning = process.env.NO_CONSTELLATION_WARNING
```

Comment:
All new environment variables should be added in `packages/wrangler/src/environment-variables/factory.ts`.

Reply:
Done.